tests/test_host/test_config_verification.py

```diff
@@ -11,11 +11,11 @@ def test_no_config(self):
 
     def test_no_evaluation(self):
         with self.assertRaisesRegex(ValueError, "Evaluation field must contain"):
-            load_config("tests/test_configs/missing_eval.json")
+            load_config("tests/data/configs/broken/missing_eval.json")
```

Comment:
Since these are host tests, can you use Pathlib to ensure that they are not OS-dependent?